test/router_test.exs

```diff
@@ -102,6 +102,32 @@ defmodule BeeBee.RouterTest do
     end
   end
 
+  describe "PUT /_update" do
+    test "updates a given short tag" do
+      :post
+      |> conn("/_add", %{"url" => "https://github.com", "short_tag" => "grape"})
+      |> Router.call([])
+
+      conn =
+        :put
+        |> conn("/_update", %{"url" => "https://github.com", "short_tag" => "grape"})
```

Comment:
It would probably be more useful to use a different url for the update and test to see that it gets updated.